src/aap_eda/core/models/queue.py

```diff
@@ -23,8 +23,14 @@ class Meta:
         ordering = ["id"]
 
     request = models.TextField(null=False, choices=ActivationRequest.choices())
+    process_parent_fqcn = models.TextField(
+        null=True, default="aap_eda.core.models.Activation"
+    )
+    process_parent_id = models.BigIntegerField(null=True)
```

Comment:
@bzwei Should we allow NULLs can we use the same migration technique as in PR #634 which sets all the data in the new columns and leaves the old one in place.

Reply:
yes, we can